diff --git a/keywords/_formatLimit.js b/keywords/_formatLimit.js
index e731a8a00a488598e56cb7cdafe2e4f9897f961b..903a9cd8574b2b85cdf0e38cc48ea5770febdd76 100644
--- a/keywords/_formatLimit.js
+++ b/keywords/_formatLimit.js
@@ -59,6 +59,7 @@ module.exports = function (minMax) {
 
 function extendFormats(ajv) {
   var formats = ajv._formats;
+  if (!formats) return;
   for (var name in COMPARE_FORMATS) {
     var format = formats[name];
     // the last condition is needed if it's RegExp from another window
